src/rsvc/srv.c

```diff
@@ -114,18 +114,27 @@ alloc_init(enum ds_rsvc_class_id class, d_iov_t *id, uuid_t db_uuid,
 	}
 
 	if (rsvc_class(class)->sc_map_dist != NULL) {
+		rc = ABT_mutex_create(&svc->s_map_dist_mutex);
+		if (rc != ABT_SUCCESS) {
+			DL_ERROR(rc, "%s: failed to create map_dist_mutex", svc->s_name);
+			rc = dss_abterr2der(rc);
```

Comment:
```suggestion
			rc = dss_abterr2der(rc);
			DL_ERROR(rc, "%s: failed to create map_dist_mutex", svc->s_name);
```
conversion from ABT error number to daos to make DL_ERROR print the daos error properly?